app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -110,8 +110,11 @@ def params_for_update
   end
 
   def self.params_for_create(provider)
-    capabilities = provider.capabilities.map do |capability|
-      {:label => "#{capability['abstract_capability']}: #{capability['value']}", :value => capability['uuid']}
+    capabilities = provider.capabilities.flat_map do |capability|
+      capability_name = capability.first
+      capability.last.map do |c|
```

Comment:
When you loop over a hash you can do:
```
{:a => "a", :b => "b"}.each do |key, val|
```
so that you don't have to do `capability.first` and `capability.last`

```suggestion
    capabilities = provider.capabilities.flat_map do |name, capability|
      capability.map do |c|
```